docs/resources/tutorials/core/qos/mutable/vulcanexus_change_mutable_qos/src/change_mutable_qos_publisher.cpp

```diff
@@ -0,0 +1,111 @@
+// Copyright 2016 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @file change_mutable_qos_publisher.cpp
+ */
+ 
+ #include "rclcpp/rclcpp.hpp"
+#include "std_msgs/msg/string.hpp"
+
+#include "rmw_fastrtps_cpp/get_participant.hpp"
+#include "rmw_fastrtps_cpp/get_publisher.hpp"
+
+#include <fastdds/dds/publisher/qos/DataWriterQos.hpp>
+
+using namespace std::chrono_literals;
+
+class Node_ChangeMutableQoS_Publisher : public rclcpp::Node
+{
+public:
+  Node_ChangeMutableQoS_Publisher(std::string node_name_prefix, std::string node_name)
```

Comment:
It is done this way because the prefix is used to conform both the name of the node (on construction and for some prints) and the name of the parameter.
Do you want me to change it in some way?